chia/cmds/start_funcs.py

```diff
@@ -59,7 +59,7 @@ async def create_start_daemon_connection(root_path: Path, config: Dict[str, Any]
     return None
 
 
-async def async_start(root_path: Path, config: Dict[str, Any], group: str, restart: bool) -> None:
+async def async_start(root_path: Path, config: Dict[str, Any], restart: bool, group: tuple) -> None:
```

Comment:
Is the re-order important?  I wonder if the hinting corrections should be submitted separately.

Reply:
I agree that we need a strong justification for reordering these. It looks unncecessary.
Also, I seem to recall the `group` parameter had a more precise tuple type in a different place. Like `Tuple[str, ...]`

Reply:
> Is the re-order important? I wonder if the hinting corrections should be submitted separately.

yeah i can undo the re-order